src/go/rdctl/pkg/snapshot/manager.go

```diff
@@ -13,9 +13,11 @@ import (
 	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
 )
 
-var nameRegexp = *regexp.MustCompile("^[0-9a-zA-Z_-]{0,100}$")
+const maxNameLength = 250
+const nameDisplayCutoffSize = 30
+
+var isWhitespaceRegexp = regexp.MustCompile(`\s`)
```

Comment:
Why are you not using `unicode.IsSpace()`? You only seem to be checking single runes anyways, and are not looking to match strings that have a space character anywhere.

Reply:
Good point